src/clarity-angular/datagrid/datagrid-cell.spec.ts

```diff
@@ -3,31 +3,33 @@
  * This software is released under MIT license.
  * The full license information can be found in LICENSE in the root directory of this project.
  */
-import {Component} from "@angular/core";
-import {TestContext} from "./helpers.spec";
-import {DatagridCell} from "./datagrid-cell";
-import {DatagridRenderOrganizer} from "./render/render-organizer";
+import { Component } from "@angular/core";
+import { TestContext } from "./helpers.spec";
+import { DatagridCell } from "./datagrid-cell";
+import { DatagridRenderOrganizer } from "./render/render-organizer";
+import { HideableColumnService } from "./providers/hideable-column.service";
 
-export default function(): void {
-    describe("DatagridCell component", function() {
+export default function (): void {
+    describe ("DatagridCell component", function () {
         let context: TestContext<DatagridCell, SimpleTest>;
 
-        beforeEach(function() {
-            context = this.create(DatagridCell, SimpleTest, [DatagridRenderOrganizer]);
+        beforeEach (function () {
+            context = this.create (DatagridCell, SimpleTest, [DatagridRenderOrganizer, HideableColumnService]);
         });
 
-        it("projects content", function() {
-            expect(context.clarityElement.textContent.trim()).toMatch("Hello world");
+        it ("projects content", function () {
+            expect (context.clarityElement.textContent.trim ()).toMatch ("Hello world");
         });
 
-        it("adds the .datagrid-cell class to the host", function() {
-            expect(context.clarityElement.classList.contains("datagrid-cell")).toBeTruthy();
+        it ("adds the .datagrid-cell class to the host", function () {
+            expect (context.clarityElement.classList.contains ("datagrid-cell")).toBeTruthy ();
```

Comment:
Sorry, IntelliJ was mis-configured for `Reformat Code` and I didn't catch this file when I cleaned up. Will fix.